modules/olmv1-catalog-queries.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-catalog-queries_{context}"]
+= Common catalog queries
+
+You can query catalogs by using the `jq` CLI tool.
+
+// https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator
+// https://docs.asciidoctor.org/asciidoc/latest/tables/data-format/#escape-the-cell-separator
+.Common package queries
+[options="header",cols="1,3"]
+|===
+|Query |Request
+
+|Available packages in a catalog
+a|
+[source,terminal]
+----
+$ jq -s '.[] \| select( .schema == "olm.package") \| \
+  .name' <catalog_name>.json
+----
+
+|Package metadata
+a|
+[source,terminal]
+----
+$ jq -s '.[] \| select( .schema == "olm.package") \| \
+  select( .name == "<package_name>")' <catalog_name>.json
+----
+
+|Catalog blobs in a package
+a|
+[source,terminal]
+----
+$ jq -s '.[] \| select( .package == "<package_name>")' \
+  <catalog_name>.json
+----
+
+|===
+
+.Common channel queries
+[options="header",cols="1,3"]
+|===
+|Query |Request
+
+|Channels in a package
+a|
+[source,terminal]
+----
+$ jq -s '.[] \| select( .schema == "olm.channel" ) \| \
+  select( .package == "<package_name>") \| .name' \
+  <catalog_name>.json
+----
+
+a|
+* Package bundles
```

Comment:
Following up from the meeting yesterday - I would move this ("Package bundles") to the common bundle queries, because while combining all the channel information together you can get a list of _likely_ all the bundles in a package it's not guaranteed. The `jq` is very similar to get that info though and would be:
```
$ jq -s '.[] \| select( .schema == "olm.bundle" ) \| \
  select( .package == "<package_name>") \| .name' \
  <catalog_name>.json
```

Reply:
Thank you, @everettraven! Feedback applied. PTAL.